.circleci/config.yml

```diff
@@ -606,11 +610,11 @@ workflows:
       # Building & packaging
 
       - build-x86_64-unknown-linux-gnu-archive:
-          <<: *require-checks-and-tests
```

Comment:
I dropped the test dependency on these so that they can run in parallel with the tests, reducing the overall CI execution time.